models/db_pull.js

```diff
@@ -12,7 +12,7 @@ function DBPull(pull) {
       state: pullData.state,
       title: pullData.title,
       body: pullData.body,
-      draft: pullData.draft,
+      draft: pullData.draft === 1,
```

Comment:
The comment here says this takes in API data and should return DB data. So I'd expect this to be `pullData.draft ? 1 : 0` instead.

Reply:
Resolved via https://github.com/iFixit/pulldasher/pull/329/commits/54ae9a6d38c471c2e3bd45a0dfd5c288f4357fec